Allow custom additions to general DSN via environment variable

dylan-tock opened the pull request:
As noted in [issue 86](https://github.com/departurerb/departure/issues/86), there is a bug in `pt-online-schema-change` where the `-P` option does not populate the port information for the general DSN which results in an "Use of uninitialized value" error.

This PR adds the ability to use "PERCONA_ADDED_DSN_ITEMS" environment variable to append items to the generated DSNs and, thereby, work around the existing bug with `pt-online-schema-change`.

dylan-tock commented:
I have tested this modification locally and it does what is expected. However, I did testing with a remote DB/username and not with a local one. Because of this and, what I *think* might be underlying assumptions with the spec tests, `rake spec` did not run cleanly for me on all tests.  However, the migration tests related to what I was trying to test passed.

benlangfeld commented on lib/departure/dsn.rb:
```diff
@@ -9,16 +9,17 @@ class DSN
     def initialize(database, table_name)
       @database = database
       @table_name = table_name
+      @added_dsn_items = ENV.fetch('PERCONA_ADDED_DSN_ITEMS', nil)
```
What do you think about this alternative phrasing?

```suggestion
      @suffix = ENV.fetch('PERCONA_DSN_SUFFIX', nil)
```

dylan-tock replied:
I knew that variable was clunky and am happy to update it to something less clunky.  I'll update this when I add a test and CHANGELOG entry (per your other comment).

benlangfeld reviewed (changes requested):
Could you please add some [test coverage](https://github.com/departurerb/departure/blob/master/spec/departure/dsn_spec.rb) and a [CHANGELOG entry](https://github.com/departurerb/departure/blob/master/CHANGELOG.md?plain=1#L7)?

dylan-tock commented:
> Could you please add some [test coverage](https://github.com/departurerb/departure/blob/master/spec/departure/dsn_spec.rb) and a [CHANGELOG entry](https://github.com/departurerb/departure/blob/master/CHANGELOG.md?plain=1#L7)?

Added these and updated the variable names per your suggestion.

Please let me know if there are any additional adjustments that need to be made.
